rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,194 @@
+    For the initial implementation we'll likely have a few subtypes of Person but not of Subject.
+
+    The alternative is to have `type: Concept` and do subtypes through something similar to workType/format, but I'm not sure that adds anything.
+
+    This is a value users would be able to filter on (if we provide an endpoint for listing/searching concepts).
+
+-   The `description` should provide enough context to help users distinguish subjects/people with the same label but different meaning.
+    This will likely be drawn from LCSH or Wikidata initially, but at some point we might write our own content here.
+
+    Q: Is `description` the best name for this field, or should it be called something different?
+
+-   The `urls` list points to this subject/person elsewhere.
+    We aren't trying to replicate LCSH/MESH/Wikidata; if somebody wants to read more in-depth about a given concept, we should link them to an external entry.
+
+    In works we use digital locations in `work.items.locations` to link to external resources.
+    We don't want to reuse that structure, which is more complicated than we need here.
+
+    Q: Aside from the URL, what other values do we want for these links?
+    e.g. link label
+
+    Q: Do we need these values at all?
+    If we put the Wikidata ID in the list of identifiers, do we leave clients to construct a URL, or do we provide it in the API?
+
+-   The `date`/`place` information is somewhat speculative.
+
+    We think we might include birth/death information on per-person pages; we could also link to significant dates and places and use that as a way to link between concepts.
+    This is similar to production events on works.
+
+    We can start with a `birthDate/birthPlace/deathDate/deathPlace` field on the initial version of the API, and decide if we want to put this information on per-person pages.
+    If yes, we can think about how to model this properly.
+    If no, we can scrap all of these fields.
+
+-   The `relatedConcepts` field lets us link between concept pages.
+    It contains a simplified subset of a Concept's fields; enough to render a link but nothing more.
+
+    We don't want to describe how two concepts are related; this is significantly more complicated.
+
+    Q: Is relatedness symmetric?
+    If `A.relatedConcepts = [B]`, is it true that `B.relatedConcepts = [A]`?
+    This might affect if/how we build an API for listing concepts.
+
+
+
+## Endpoint #2: Concepts in the works API
+
+We'll simplify how concepts appear in the works API.
+We'll provide enough information to render a link to the per-person or per-subject page, but nothing more -- for detailed information, use the concepts API.
+
+For example:
+
+```
+GET /works?contributors.agent=azxzhnuh
+{
+  …,
+  "contributors": [
+    {
+      "agent": {
+        "id": "azxzhnuh",
+        "label": "Florence Nightingale",
+        "type": "Person"
+      },
```

Comment:
I think this implies that the works API knows that `Person`s and `Organisation`s (for example) are `agent`s, even when the concepts API doesn't. I think that's a good thing.

Reply:
Agreed, and this has a subtle effect on how we might filter for agents – I've added that to the notes on `type`.